components/RecordingsList.vue

```diff
@@ -50,6 +50,9 @@ export default {
   },
   methods: {
     remove(id) {
+      if (!confirm('Are you sure you want to remove this record?')) {
+        return
+      }
```

Comment:
Thanks for your contribution @simonjcarr! Much appreciated :) 

We have an issue (#5 ) related to this. I will update it to describe better the approach I would like to take.

> Vuetify has [a dialog component](https://vuetifyjs.com/en/components/dialogs/#modal) that we can use in this case.

Wanna use that instead?

Reply:
I have submitted a new PR using a Vuetify dialog box.